examples/PolyOseen.cc

```diff
@@ -0,0 +1,1435 @@
+
+            for (unsigned int f = 0; f < 4; ++f)
+              if (!cell->at_boundary(f))
+                {
+                  if ((cell->face(f)->center()[0] > 0.1) &&
+                      (cell->face(f)->center()[0] < 0.9))
+                    {
+                      if (cell->face(f)->center()[1] > 1.)
+                        cell->face(f)->set_all_manifold_ids(1);
+                      else
+                        cell->face(f)->set_all_manifold_ids(2);
+                    }
+
+                  if ((cell->face(f)->center()[1] > 0.6) &&
+                      (cell->face(f)->center()[1] < 1.4))
+                    {
+                      if (cell->face(f)->center()[0] > 0.5)
+                        cell->face(f)->set_all_manifold_ids(3);
+                      else
+                        cell->face(f)->set_all_manifold_ids(4);
+                    }
+                }
+          }
+        triangulation.refine_global(6);
+
+        triangulation.save(filename);
+        std::cout << "     Saved grid to " << filename << std::endl;
+
+        GridOut       grid_out;
+        std::ofstream out("base_grid_Oseen.msh");
+        grid_out.write_msh(triangulation, out);
+      }
+  }
+
+  /*============================================================================
+    Generate an agglomerated mesh using R3MG.
+
+    Builds a fine background grid, partitions each material subdomain
+    using separate R-trees, and performs cell agglomeration independently.
+  ============================================================================*/
+  template <int dim>
+  void
+  OseenProblem<dim>::make_agglo_grid()
+  {
+    make_base_grid();
+
+    std::cout << "     Size of base grid: " << triangulation.n_active_cells()
+              << std::endl;
+    cached_tria =
+      std::make_unique<GridTools::Cache<dim>>(triangulation, mapping);
+    agglo_handler = std::make_unique<AgglomerationHandler<dim>>(*cached_tria);
+
+    // Partition with Rtree
+    namespace bgi = boost::geometry::index;
+    static constexpr unsigned int max_elem_per_node =
+      PolyUtils::constexpr_pow(2, dim);
+
+    std::vector<
+      std::vector<std::pair<BoundingBox<dim>,
+                            typename Triangulation<dim>::active_cell_iterator>>>
+      all_boxes(num_domain);
+    // To preserve the curved boundaries of the domains, we use separate R-trees
+    // for each domain. A "boxes" is a collection of bounding boxes for a single
+    // domain. "all_boxes" is a collection of all such "boxes".
+    for (const auto &cell : triangulation.active_cell_iterators())
+      all_boxes[cell->material_id()].emplace_back(
+        mapping.get_bounding_box(cell), cell);
+
+    for (unsigned int i = 0; i < num_domain; ++i)
+      {
+        auto tree = pack_rtree<bgi::rstar<max_elem_per_node>>(all_boxes[i]);
+
+        std::cout << "     Total number of available levels in domain_" << i << ": "
+                  << n_levels(tree) << std::endl;
```

Comment:
The indenter is complaining about this line. Can you fix it as follows?
```suggestion
        std::cout << "     Total number of available levels in domain_" << i
                  << ": " << n_levels(tree) << std::endl;
```